### ISSUE TITLE
inconsistency with javascript template literals

### ISSUE THREAD

**Issue by obedm503** [akamud/vscode-theme-onedark#76]
nested string delimiter colors inside template interpolation are red instead of greenish. happens with single quotes, double quotes and backticks.

## Sample code

```js
const someString = 'this a string';

const someTemplateLiteral = `this is a template literal`;

const nestedTemp = `template string ${ 'normal string' } nested ${ `another string ${ 'even deeper' }` }`

```

## Filename and Language selected

using `.js` extension with javascript file type selected
## Atom Original theme Screenshot

![atom](http://i.imgur.com/OSPuFwE.png)

## VSCode theme Screenshot

![code](http://i.imgur.com/b0uBWXd.png)

## Versions used

* VSCode version: Version 1.16.1 Commit 27492b6bf3acb0775d82d2f87b25a93490673c6d
* Theme version: 2.0.5

# Extra information
found it interesting that vscode with the theme highlights consistently across js and ts files, but atom seems to not recognize template literals in typescript files at all. 
if you need any more screenshots let me know.

**Comment by akamud** [comment]
Thank you for taking the time reporting this. I will let you know if this is fixable.

Indeed, Atom coloring for TS is very inconsistent, I try to always add the same coloring for JS and TS.

**Comment by akamud** [comment]
This will be fixed in the next release, 2.0.6.

Thank you again for your help!
